app/templates/application.hbs

```diff
@@ -58,3 +58,14 @@
 
   </div>
 </div>
+
+{{!--
+  Display a site-wide nothification with the site-message component:
+  @message - string: conditionally displays component
+  @warning - boolean: adds an exclamation-triangle icon
+  @dismissible - boolean: allows user to close message
+--}}
+{{site-message
+  warning=true
```

Comment:
completely optional nit: maybe we just set the defaults to true, and remove these arguments? I don't know when we would ever want a non-dismissible popup anyway.

Reply:
that's a good point, will do